app/2.0/docs/devguide/dom-template.md

```diff
@@ -60,106 +59,183 @@ except for testing. For caveats about defining elements in the main document,
 see [main document definitions](registering-elements#main-document-definitions).
 {.alert .alert-info}
 
-### Specify a string template
+### Provide a template by defining a template property on the constructor {#templateobject}
 
-As an alternative to specifying the element's template in markup, you can specify a string template
-by creating a static `template` getter that returns a string.
+As an alternative to specifying the element's template in markup, you can define a static `template`
+property. The `template` property must be an `HTMLTemplateElement`.
```

Comment:
must be => should be

This section is correct for 3.x, but for 2.x we support returning a string, as well.

Maybe in place of the second sentence, add something to the next para, like,

In Polymer 2.x, the template can be supplied as an `HTMLTemplateElement` instance or a string. Support for returning a string template is being discontinued in Polymer 3.x. Starting in Polymer 2.4 <<<verify actual release when it lands>>>, Polymer provides ... This is the recommended way of defining a string template in Polymer 2.4 and later.